docs/src/Guides/01 Getting Started.md

```diff
@@ -56,12 +56,13 @@ Now let's get the library installed.
 Now let's get a basic bot going, for your code, you'll want something like this:
 
 ```python
-from interactions import Client, Intents, listen
+from interactions import Client, listen
 
-bot = Client(intents=Intents.DEFAULT)
-# intents are what events we want to receive from discord, `DEFAULT` is usually fine
+bot = Client(
+    token="Put your token here"
+)
 
-@listen()  # this decorator tells snek that it needs to listen for the corresponding event, and run this coroutine
+@listen()  # listen for the corresponding event, and run this coroutine
```

Comment:
Now thinking about this, this should probably be changed to a `@listen(Ready)` to match up with the rest of the docs. Good catch with the comments though.